src/win32_surface.rs

```diff
@@ -86,6 +101,6 @@ impl Deref for Win32Surface {
 
 impl Clone for Win32Surface {
     fn clone(&self) -> Win32Surface {
-        unsafe { from_glib_none(self.to_glib_none().0) }
+        unsafe { Self::from_raw_none(self.to_raw_none()) }
```

Comment:
Either way would work here, I think. The way it is now is closer to the existing code. Calling clone directly would be more direct and consistent with `image_surface.rs`. If it were up to me I'd probably change it to call clone and get rid of the `from_raw_none` implementation.